### ISSUE TITLE
Unable to navigate past text input boxes in JAWS using up arrow key

### ISSUE THREAD

**Issue by PrinceMinhas** [carbon-design-system/carbon#3804]
**Env**

Windows 10
Google Chrome
JAWS

**Detailed description**

I am currently getting an accessibility issue regarding the carbon text input boxes I believe. When using JAWS screen reader, and I navigate to the bottom of the page or modal (using down arrow), and then I try to go back to the top using the up arrow I can not move past the last text input box on the page. Whether or not the input box is populated or not, the up arrow does not change the screen reader focus past the text input box. https://codepen.io/team/carbon/full/MRyeVV/, to reproduce you can try to play with the carbon modals. Using *only* the arrow keys navigate to the bottom and try to go back to the top. Thanks.

**Comment by dakahn** [comment]
@snidersd

**Comment by dakahn** [comment]
@snidersd I can recreate in Firefox on Windows 10 with JAWS 2019 (September build). It does look like keyboard commands for reading can be used to navigate down through the modal, but not _all the way back up_. 

That said, `tab` and `shift+tab` can still be used to access all elelements inside the modal in the appropriate order and `esc` still works to close the modal altogether. 

Downgrading severity, but still investigating further.

**Comment by dakahn** [comment]
A clue maybe -- when using tab to navigate between focusable elements in the modal we get what we expect: close modal button `tab` --> text input `tab` --> secondary button `tab` --> primary button

But starting at the top of the modal and using `down arrow` to read through the modal the order is strange. JAWS doesn't seem to be interpreting our DOM order here correctly because elements aren't being read in a logical order (primary button coming after secondary button even though they're flipped in the DOM).

**Comment by dakahn** [comment]
attn @snidersd 

[So it looks like this is a known bug with JAWS](https://github.com/FreedomScientific/VFO-standards-support/issues/199#issuecomment-521641212)

It also _might not be a bug_ and be working as intended? They may want you to hit `num +` to exit Forms mode and use the virtual cursor from then on. I changed the priority based on this information. [See this comment](https://github.com/FreedomScientific/VFO-standards-support/issues/199#issuecomment-521641212)

**Comment by dakahn** [comment]
Closing for now, but we can reopen if we need to.
